interop-tests/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# syntax=docker/dockerfile:1.5-labs
+FROM rust:1.67.0
```

Comment:
Can this be a multi-stage dockerfile?

Reply:
Sure, but why? We aren't pushing this to a registry or uploading/downloading the image somewhere.

Reply:
Right and we need to pull all layers anyway so I guess it doesn't matter!